common/views/components/PageHeader/HighlightedHeading.tsx

```diff
@@ -7,7 +7,7 @@ const Heading = styled(Space)`
   display: block;
   background-color: ${props => props.theme.color('white')};
 
-  @supports (box-decoration-break: clone) {
+  @supports (-webkit-box-decoration-break: clone) {
```

Comment:
Right, so enabling vendor prefixes only does it for the styles and not for the selectors, so it had to be done manually. Another reason to disable them and hand write them?